zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/monitoring/BackupMXBean.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.server.backup.monitoring;
+
+public interface BackupMXBean {
+  // Snapshot backup metrics
+
+  /**
+   * @return Number of snapshot backup errors since last successful snapshot backup iteration
+   */
+  int getSnapshotErrorCount();
```

Comment:
This is confusing - is this # of errors for the last snapshot backup iteration? 

Suppose a snapshot backup iteration A fails, and it somehow continues to fail - as in B, C, D, E... all fail. What kind of value do we expect to see coming out of this metric?

Reply:
This number of error is the accumulative count of errors since last successful backup iteration.
So if A fails, the number is 1; if then B succeeds, the number is reset to 0. If A fails, the number is 1; if then B fails too, the number increments to 2.